style.css

```diff
@@ -528,20 +926,808 @@ li textarea {
+    position: absolute;
+    width: 124px;
+    height: 48px;
+    margin-left: 0;
+    margin-top: 54px;
+    top: calc(50% - 48px / 2);
+    background: #fff;
+    border: 1px solid #7f8cff;
+    box-sizing: border-box;
+    border-radius: 8px;
+    order: 3;
+  }
+
+  .action {
+    position: static;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 24px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    letter-spacing: 0.03em;
+    color: #6070ff;
+    flex: none;
+    flex-grow: 0;
+    text-decoration: none;
+  }
+
+  .btn:hover {
+    background: #2230d2;
+    border: 1px solid #7f8cff;
+  }
+
+  .btn:active {
+    background: #0416b6;
+    color: white;
+  }
+
+  .btn:visited {
+    background: #2230d2;
+    border: 1px solid #7f8cff;
+  }
+
+  .btn:disabled {
+    background: #e5e5e5;
+    border: 1px solid;
+  }
+
+  .action:hover {
+    color: #fff;
+  }
+
+  .action:visited {
+    color: #fff;
+  }
+
+  .action:disabled {
+    color: #5e6c84;
+  }
+
+  .work-bottom {
+    opacity: 0;
+  }
+
+  .bg-modal {
+    background-color: rgba(193, 199, 208, 0.8);
+    width: 100%;
+    height: 100%;
+    position: fixed;
+    top: 8%;
+    display: none;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .modal-content {
+    width: 65%;
+    height: 90%;
+    background-color: #fff;
+    border: 1px solid #dfe1e6;
+    box-sizing: border-box;
+    border-radius: 16px;
+    padding: 16px;
+  }
+
+  .modal-content .frame {
+    padding-left: 20px;
+  }
+
+  .modal-content .snapshotdesktop1 {
+    width: 70%;
+    height: 50%;
+    display: block;
+    margin: 10px auto;
+  }
+
+  .modal-header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding-left: 20px;
+  }
+
+  .btm-content {
+    display: flex;
+    align-items: baseline;
+    margin-top: 2%;
+  }
+
+  .descr-1 {
+    width: 70%;
+  }
+
+  .close-button {
+    transform: rotate(45deg);
+    cursor: pointer;
+    font-size: 40px;
+    color: grey;
+  }
+
+  .bg-modal .primary-text2 {
+    width: 100%;
+  }
+
+  .buttons {
+    display: flex;
+    gap: 10px;
+    justify-content: center;
+    margin-top: 13%;
+  }
+
+  .live-btn {
+    background-color: #fff;
+    color: #396df2;
+    padding: 10px 4px;
+    font-family: 'Poppins', sans-serif;
+    font-size: 16px;
+    border: 1px solid #6070ff;
+    box-sizing: border-box;
+    border-radius: 8px;
+    cursor: pointer;
+    height: 50px;
+    width: 50%;
+    margin-top: 0.5%;
+  }
+
+  .about {
+    position: relative;
+    display: grid;
+    grid-template-columns: 40% 60%;
+    grid-column-gap: 80px;
+    align-items: center;
+    box-sizing: border-box;
+    margin: 142px 0 10px 0;
+    background: url("images/abt-myself-desktop.png") no-repeat;
+    background-size: cover;
+    border-top-right-radius: 70px;
+    top: 2460px;
+    width: 100%;
+    height: 924px;
+  }
```

Comment:
- [ ] I think it would be better to add a **space above the ```about``` section** to separate it to the ```project``` section as this will help us to match the Figma design :+1: :+1:  You may add a ```margin-top``` to fix this issue
- [ ] It would be better also to add the ```padding: 142px;``` on the about section to match that of Figma design. :+1: :+1: 

**Your design**
![image](https://user-images.githubusercontent.com/102757126/200083884-a8538c5b-c085-489b-9141-9bdd48511751.png)